### DISCUSSION TITLE
i dont use my rial version

### DISCUSSION THREAD

**Discussion by JuanGaliana** [community/community#38246] [Copilot Conversations]
Hello, I have a one-year subscription whose payment I have to make on December 25. Now I have the free version enabled but I can't use it, it tells me that it has expired and therefore I can't use copilot.
Thank you
All the best

**Comment by iamcjwilliams** [comment]
@JuanGaliana Sorry for the trouble here. The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted :)
